src/IdentityServer/src/IdentityServer.Core/Events/IdOpsEventSink.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Duende.IdentityServer.Events;
+using Duende.IdentityServer.Services;
+using IdOps.IdentityServer.Extensions;
+using Microsoft.Extensions.Logging;
+
+namespace IdOps.IdentityServer.Events
+{
+    public class IdOpsEventSink : IEventSink
+    {
+        private readonly ILogger<BusEventSink> _logger;
+        private readonly IEnumerable<IIdOpsEventSink> _sinks;
+
+        public IdOpsEventSink(ILogger<BusEventSink> logger, IEnumerable<IIdOpsEventSink> sinks)
+        {
+            _logger = logger;
+            _sinks = sinks;
+        }
+
+        public Task PersistAsync(Event evt)
+        {
+            try
+            {
+                Task.Run(() => ProcessAsync(evt)).Forget();
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex, "Error while sending event");
+            }
+
+            return Task.CompletedTask;
+        }
+
+        private async Task ProcessAsync(Event evt)
+        {
+            foreach (IIdOpsEventSink sink in _sinks)
```

Comment:
Do we want to process this in sequence or extract a method and do Task.WhenAll()?

Reply:
I think it's ok like this for the moment, we have to take a look maybe also to the forget action